### PR TITLE
[Fix] Added padding to percentage string

### PR DESCRIPTION
**Pull request by olivernybroe**
This PR add's padding to the percentage string so the length is always 5

resolves: #22

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,9 @@
         "roave/no-floaters": "^1.1",
         "phpstan/phpstan": "^0.11.5",
         "symfony/var-dumper": "^4.2",
-        "thecodingmachine/phpstan-strict-rules": "^0.11.0"
+        "thecodingmachine/phpstan-strict-rules": "^0.11.0",
+        "phpunit/phpunit": "^7.5|^8.0",
```
Comment (nunomaduro) [inline composer.json]: I would bump this phpunit 8 only.

```suggestion
        "phpunit/phpunit": "^8.0",
```

File: src/Application/Console/Style.php [hunk 2]
```diff
@@ -276,17 +283,19 @@ public function waitForKey(string $category): Style
...
     /**
-     * Returns the percentage as 4 chars string.
+     * Returns the percentage as 5 chars string.
      *
      * @param  float  $percentage
      *
      * @return string
      */
-    private function getPercentageAsString(float $percentage): string
+    public static function getPercentageAsString(float $percentage): string
```
Comment (nunomaduro) [inline src/Application/Console/Style.php]: I think this can be private.

File: tests/Application/Console/StyleTest.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (nunomaduro) [inline tests/Application/Console/StyleTest.php]: Could you add strict type, plus final casses on tests too? Sorry, I am crazy.

File: tests/Application/Console/StyleTest.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+use NunoMaduro\PhpInsights\Application\Console\Style;
+use Tests\TestCase;
+
+/**
+ * @covers \NunoMaduro\PhpInsights\Application\Console\Style
+ */
+class StyleTest extends TestCase
+{
+    /** @test */
+    public function string_has_correct_length_when_one_digit_value()
```
Comment (nunomaduro) [inline tests/Application/Console/StyleTest.php]: Could we use the style testStringHasCorrectLengthWhenOneDigitValue? Do you mind?

Comment (nunomaduro) [reply tests/Application/Console/StyleTest.php]: I would add return `void` too.

Comment (nunomaduro) [reply tests/Application/Console/StyleTest.php]: Same goes for the other tests.

File: tests/Application/Console/StyleTest.php (hunk 3 above)
Comment (nunomaduro) [reply tests/Application/Console/StyleTest.php]: Same goes for the other classes tests too.

File: src/Application/Console/Style.php (hunk 2 above)
Comment (nunomaduro) [reply src/Application/Console/Style.php]: On testing, you may need to use reflection

**Comment by olivernybroe** [comment]
Should have gotten around your comment :) 
Just say if you have any other changes you want me to do.

**Comment by nunomaduro** [comment]
Thanks! Great job!

**Pull request merged**